tljh/installer.py

```diff
@@ -508,7 +478,6 @@ def main():
     logger.info("Setting up JupyterHub...")
     ensure_node()
     ensure_jupyterhub_package(HUB_ENV_PREFIX)
-    ensure_jupyterlab_extensions()
```

Comment:
Do we also need to keep `Node.js`?

I believe it was required to be able to install JupyterLab extensions?

Reply:
Otherwise if we keep nodejs, we'll have to update it to version 12 at least (better would be 14), since JupyterLab 3.0 requires nodejs 12+